site/src/hooks/useOrganizationId.ts

```diff
@@ -1,13 +1,6 @@
-import { useAuth } from "components/AuthProvider/AuthProvider";
-import { isAuthenticated } from "xServices/auth/authXService";
+import { useMe } from "./useMe";
 
 export const useOrganizationId = (): string => {
-  const [authState] = useAuth();
-  const { data } = authState.context;
-
-  if (isAuthenticated(data)) {
-    return data.user.organization_ids[0];
-  }
-
-  throw new Error("User is not authenticated");
+  const user = useMe();
+  return user.organization_ids[0];
```

Comment:
This is something that would be caught with some of TS's stricter compiler settings: as far as TypeScript's concerned, it can't guarantee that an array has content in it when you access it at index 0

Do you think it'd be worth adding a check in here to throw an error if `user.organization_ids[0]` ever evaluates to `undefined`?

Reply:
I think there is a TS config for that but I would not enable it right now, since probably triggers errors in other places. In this case, a user always will have one organization so we don't need to worry about it right now.

Reply:
Yeah, turning the setting (`noUncheckedIndexedAccess`) on is something I want to do eventually, but I agree – right now isn't the right time